pkgs/development/python-modules/ajsonrpc/default.nix

```
@@ -0,0 +1,22 @@
+{ lib, isPy3k, buildPythonPackage, fetchPypi, pytestCheckHook }:
+
+buildPythonPackage rec {
+  pname = "ajsonrpc";
+  version = "1.1.0";
+
+  disabled = !isPy3k;
```

```suggestion
  disabled = pythonOlder "3.5";
```

see https://github.com/pavlov99/ajsonrpc/blob/master/setup.py#L32